client/src/Components/Feelings/feelingsData.js

```diff
@@ -3,23 +3,39 @@ const feelings = [{
     'associatedFeelings': [
       'Happy',
       'Content',
-      'Optimistic'
+      'Optimistic',
+      'Powerful',
+      'Confident',
+      'Inquistive',
+      'Hopeful',
+      'Inspired',
+      'Playful',
+      'Proud',
+      'Thankful',
+      'Valued',
+      'Free',
+      'Centered'
     ]
   },
   {
     'feeling': 'Sad',
     'associatedFeelings': [
       'Sad',
       'Lonely',
-      'Depressed'
+      'Depressed',
+      'Empty',
+      'Ashamed'
     ]
   },
   {
     'feeling': 'Surprised',
     'associatedFeelings': [
       'Surprised',
       'Confused',
-      'Excited'
+      'Excited',
+      'Amazed',
+      'Energetic',
+      'Encouraged'
```

Comment:
Crying rn, love your thoughtfulness to our user.

Reply:
So thoughtful chrissy!!!!!